check-versions.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2020 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+# exit when any command fails
+set -e
+
+# log aio version
+aio info
+
+# log imagemagick version
+echo "....................................................................................................."
+identify --version
+echo "....................................................................................................."
```

Comment:
Add also other tools we use, like `exiftool`, so we have everything.

Reply:
I couldn't figure out how to get the exiftool version so the best I did was run exiftool on a file to check its installed: https://github.com/adobe/asset-compute-sdk/blob/019e4e48262fd04f30f0d9e2ff2c33cd47e19092/check-versions.sh#L24-L27 (just running `exiftool` or `exiftool -v` or `exiftool --version`  logs waaay to much info. it logs the entire documentation)

Reply:
ah I figured it out:
```
exiftool -ver
```